src/OrchardCore.Modules/OrchardCore.Liquid/Startup.cs

```diff
@@ -24,8 +23,6 @@ public class Startup : StartupBase
         static Startup()
         {
             TemplateContext.GlobalMemberAccessStrategy.Register<ContentItem>();
-            TemplateContext.GlobalMemberAccessStrategy.Register<ClaimsPrincipal>();
```

Comment:
Why did you remove it, it's necessary to do `User.Identity.Name`

Reply:
Ok. I thought this would need to be done through the `O.Users` module